### DISCUSSION TITLE
Is there a way to include the score from the string similary in the results set as an additional column

### DISCUSSION THREAD

**Discussion by Freewilly3d** [moj-analytical-services/splink#177] [Q&A]
In reviewing my output I'm trying to assess whether a minor change in my string similarity calculation would provide better results.  I would be helpful if the scores could be included in the output to better see the impact of the similarity calculations on results.

**Comment by RobinL** [comment]
Thanks for the feedback.

For now, one way of doing this is to use the:
`"retain_matching_columns": True` option in your settings object.  This will output your comparison columns in the final dataset, and you could then write your string comparison function against this dataframe.
